File: Todos/C/BEE 1847.c
/*
  autor: Vinist021;
  data: 26/12/2024;
  nome: Prefácio;
*/

#include <stdio.h>

int main(int argc, char *argv[]) {
	
	int a, b;
	int q, r;
	
	scanf("%d %d", &a, &b);
	
	q = a / b;
	r = a % b;
	
	if(r < 0 && b > 0)
	{
		r += b;
		q -= 1;
	}
	else if(r < 0 && b < 0)
	{
		r -=b;
		q += 1;
	}
	
	printf("%d %d\n", q, r);

	return 0;
}

File: Todos/Python/BEE 1847.py
#   autor: Vinist021;
#   data: 25/12/2025;
#   nome: Bem-vindos e Bem-vindas ao Inverno!;

A, B, C = map(int, input().split())
PS = B - A
ST = C - B

if PS < 0 and ST >= 0:
    print(':)')
elif PS > 0 and ST <= 0:
    print(':(')
elif PS > 0 and ST > 0 and ST < PS:
    print(':(')
elif PS > 0 and ST > 0 and ST >= PS:
    print(':)')
elif PS < 0 and ST < 0 and ST > PS:
    print(':)')
elif PS < 0 and ST < 0 and ST <= PS:
    print(':(')
elif PS == 0 and ST > 0:
    print(':)')
else:
    print(':(')
